app/src/main/java/com/ajdi/yassin/netfleak/remote/api/ApiUtils.java

```diff
@@ -0,0 +1,63 @@
+package com.ajdi.yassin.netfleak.remote.api;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by @author = GHARRAS
+ * On 17/10/2019
+ * Email med.gharras@gmail.com
+ */
+public class ApiUtils {
+
+    private static final String BASE_URL = "https://api.themoviedb.org/3";
+
+    private static OkHttpClient sOkHttpClient;
+
+    private static MovieService sInstance;
+
+    private static final Object sLock = new Object();
+
+    /**
+     * because we are using static method in this class we need to initialize our OkHttpClient
+     * in the static initializer so that we want need to instantiate it each time we do a new
+     * request
+     */
+
+    {
```

Comment:
missing `static  `key word here

Reply:
either we put static or not is the same for the static block if I remember correctly that why I didn't put it

Reply:
I just pushed one with the static keyword